examples/pure/src/lib.rs

```diff
@@ -250,6 +250,48 @@ fn default_value(num: Number) -> Number {
     num
 }
 
+#[gen_stub_pyfunction]
+#[pyfunction]
+#[gen_stub(override_type(type_repr="collections.abc.Callable[[str]]", imports=("collections.abc")))]
+fn fn_override_type<'a>(
+    #[gen_stub(override_type(type_repr="collections.abc.Callable[[str]]", imports=("collections.abc")))]
+    cb: Bound<'a, PyAny>,
+) -> PyResult<Bound<'a, PyAny>> {
+    cb.call1(("Hello!",))?;
+    Ok(cb)
+}
+#[gen_stub_pyclass]
+#[pyclass]
+struct OverrideType {
+    num: isize,
+}
+
+#[gen_stub_pymethods]
+#[pymethods]
+impl OverrideType {
+    #[gen_stub(override_type(type_repr="typing_extensions.Never", imports=("typing_extensions")))]
+    fn error(&self) -> PyResult<()> {
+        Err(PyErr::new::<pyo3::exceptions::PyRuntimeError, _>(
+            "I'm an error!",
+        ))
+    }
+
+    #[getter]
+    #[gen_stub(override_type(type_repr="int", imports=()))]
```

Comment:
```suggestion
    #[gen_stub(override_type(type_repr="int"))]
```

Your implementation looks to be able to skip `imports`. This case should be included in the example.